package/MDAnalysis/analysis/rdf.py

```diff
@@ -95,6 +95,10 @@
       :class:`numpy.ndarray` of the centers of the `nbins` histogram
       bins.
 
+       .. deprecated:: 2.0.0
+           This attribute will be removed in 3.0.0.
+           Use :attr:`results.bins` instead.
+
    .. attribute:: edges
```

Comment:
Shouldn't `edges` and `count` be `results`? (I'm not super familiar with the MDA RDF classes so maybe someone like @orbeckst or @fiona-naughton can comment here)

Reply:
Yes, you need them for plotting and downstream processing.